pypeit/data/arc_lines/lists/CuI_lines.dat

```diff
@@ -1,10 +1,38 @@
-# Creation Date: 2018-Oct-04
-# Creation Date: 2018-Oct-04
-# Creation Date: 2018-Oct-04
-| ion |       wave | NIST | Instr | amplitude |                     Source |
-| CuI |  4260.6000 |    1 |    64 |        92 |  GMOS_R400_blue_1_fit.json |
-| CuI |  4481.6070 |    1 |    64 |        65 |  GMOS_R400_blue_1_fit.json |
-| CuI |  5018.0100 |    1 |    64 |        66 |  GMOS_R400_blue_1_fit.json |
-| CuI |  5144.1000 |    1 |    64 |        36 | GMOS_R400_blue_15_fit.json |
-| CuI |  5221.5230 |    1 |    64 |        31 | GMOS_R400_blue_15_fit.json |
-| CuI |  6034.0000 |    1 |    64 |       167 | GMOS_R400_blue_38_fit.json |
+# Creation Date: 2018-Nov-01
```

Comment:
I've been very careful not to add a line unless it is
for certain detected and vetted as good by one of us.

Is that true for all these lines?